packages/docs/site/docs/14-contributor-day/01-index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: WordCamp Contributor Day
+slug: /wordcamp-contributor-day
+---
+
+## Introduction
+
+The WordPress Playground VS Code extension and wp-now are tools that streamline the process of setting up a local WordPress environment. Both are powered by WordPress Playground, eliminating the need for other tools like Docker and MySQL.
+
+Read on to see how you can use either for contributing to WordPress. Please note that they’re currently in an alpha state and not all Make teams are fully supported.
+
+## Getting Started
+
+### VSCode Playground extension
+
+[Visual Studio Code Playground extension](https://marketplace.visualstudio.com/items?itemName=WordPressPlayground.wordpress-playground) offers a friendly user interface to a local WordPress environment.
+
+To get started, open your Visual Studio Code and navigate to the extensions tab. In the search bar, type 'WordPress playground' and click 'Install'. This will add a new icon on your sidebar.
+
+## wp-now
+
+### Minimum Requirements
+
+To get started with wp-now, you need to have Node and npm installed on your system. You can follow this guide to install them: [Node and npm installation guide](https://nodejs.org/en/download/package-manager). We also recommend using nvm to easily change the node version: [nvm installation guide](https://github.com/nvm-sh/nvm#installing-and-updating).
+
+### Installing
+
+You can globally install wp-now npm package in your machine, or if you prefer, you can run these commands directly from your browser using GitHub codespaces.: [https://www.npmjs.com/package/@wp-now/wp-now](https://www.npmjs.com/package/@wp-now/wp-now).
+
+```
+npm install -g @wp-now/wp-now
+```
+
+### Running
+
+To start a new server, navigate to your WordPress plugin or theme directory and run the command wp-now start. This will start the server with the default PHP and WordPress versions. A new tab will open automatically in your browser.
+
+```
+cd gutenberg
+wp-now start
+```
+
+You can stop wp-now by killing the process by pressing the keys `CTRL + C`.
+
+To change the PHP or WordPress version, you can use the `--php` and `--wp` arguments. For example, to start a server with PHP 7.4 and WordPress 6.1, you can run:
+
+```
+wp-now start --php 7.4 --wp 6.1
+```
+
+## Contributing to Gutenberg
+
+### Create a Gutenberg PR Step By Step
+
+Begin by creating a fork of the Gutenberg repository using your GitHub account. Once you've done that, clone your forked repository to your local machine to download the code. Following this, install the necessary dependencies and build the Gutenberg files.
+
+```
+git clone git@github.com:WordPress/gutenberg.git
+cd gutenberg
+npm install
+npm run dev
+```
+
+Ensure that wp-now is installed on your system. Then, open a new terminal window and start the WordPress server by executing the wp-now command.
+
+```
+cd gutenberg
+wp-now start
```

Comment:
One issue I just ran into...

My shell runs `nvm use` automatically, so I was switched to `node 14` when I opened Gutenberg:

<img width="1014" alt="image" src="https://github.com/WordPress/wordpress-playground/assets/36432/c6d47fd8-b2ae-436e-ba5f-89994cbaf702">

Reply:
Wasn't there some `wp-now` code to display a warning in this case?